test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/OpenApiHttpTriggerContextTests.cs

```diff
@@ -81,7 +82,7 @@ public async Task Given_Type_When_Initiated_Then_It_Should_Return_PackageAssembl
         public async Task Given_Type_When_Initiated_Then_It_Should_Return_OpenApiConfigurationOptions()
         {
             var location = new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.FullName;
-            var context = new OpenApiHttpTriggerContext();
+            var context = new OpenApiHttpTriggerContext(null);
```

Comment:
If we change the constructor parameter to be optional, we don't need to change this part.